tools/benchmarking/frame-weight-template.hbs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
```

Comment:
I wouldn't put the benchmarking inside the tool folder but inside its own folder

Reply:
a1549cd -- this moves benchmarking to its own root dir. Seems a bit noisy to me, especially with only one file in it now. Let me know if you dislike it.